test/data/Vanilla/Bukkit/Folia/folia-1-19-4-unsupported-plugin.json

```diff
@@ -1971,7 +1971,7 @@
     "analysis": {
         "problems": [
             {
-                "message": "The plugin motdgg does not support regional ticking.",
+                "message": "The plugin motdgg-bukkit-1.2.0 does not support regional ticking.",
```

Comment:
This should still load the plugin name and not the file name.

Reply:
Fixed